back/.rubocop.yml

```diff
@@ -20,20 +20,22 @@ Metrics/BlockLength:
   - 'db/migrate/**/*'
   # Max: 25
 Metrics/MethodLength:
-  IgnoredMethods: ['included', 'prepended', 'extended']
   Enabled: true
+  Max: 50
```

Comment:
Maybe 50 is a bit too many. I would vote for 20-30.

Reply:
Sure, also here I forgot why :)